.github/semantic.yml

```diff
@@ -0,0 +1,5 @@
+# Always validate the PR title AND all the commits
+titleAndCommits: true
```

Comment:
let's do `titleOnly`, it seems less invasive to newcomers who will typically end up with squashed commits anyhow

Reply:
- [x] Fixed in 48a65795c00df13544f0a0d60fc75c4fcb0c9f6c